examples/pong/Pong.cpp

```diff
@@ -72,7 +84,12 @@ int main()
     pauseMessage.setCharacterSize(40);
     pauseMessage.setPosition(170.f, 150.f);
     pauseMessage.setFillColor(sf::Color::White);
+    
+    #ifdef SFML_SYSTEM_IOS
```

Comment:
What about them? Do we augment the SFML API to test if touch support is enabled? Do we define a new `SFML_SYSTEM_TOUCH` macro to cover these cases? But then we have to also test if it's actually being used, because some machines will have mice AND touch screens, and some will have touch screens which emulate mice?

If you have a reasonable suggestion to handle all these hypotheticals for this PR then fill your boots. Otherwise I say we leave these decisions for if/when those cases actually affect this code (currently this example isn't used by android, and windows touch support is stuck at soon(TM))